src/Containers/Reports/Layouts/AutomationCalculator/AutomationCalculator.tsx

```diff
@@ -397,18 +392,7 @@ const AutomationCalculator: FC<AutmationCalculatorProps> = ({
           }}
         />
       ) : (
-        <EmptyState>
-          <EmptyStateIcon icon={ExclamationTriangleIcon} />
-          <Title headingLevel="h4" size="lg">
-            You have disabled all views
-          </Title>
-          <EmptyStateBody>
-            Enable individual views in the table below or press Show all button.
-          </EmptyStateBody>
-          <Button variant="primary" onClick={() => setEnabled(undefined)(true)}>
-            Show all
-          </Button>
-        </EmptyState>
+        <NoData />
```

Comment:
Try replacing this line with the code below to better match with feedback from UX. You will need to add a prop to the `EmptyList` component to allow you to add a button while also keeping the 'search' icon [(this is how I have implemented it in my WIP PR)](https://github.com/RedHatInsights/tower-analytics-frontend/pull/796/files#diff-3dba81a841e3492a20e5ab53172bb0be64e9848d66ff2ffa8f3c4611993cfe44R42). Feel free to let me know if you have any questions. 
```suggestion
        <EmptyList
          label={'Clear all filters'}
          title={'No results found'}
          message={
            'No results match the filter criteria. Clear all filters and try again.'
          }
          // addButton={true}
          path={paths.getDetails('automation_calculator')}
        />
```

Reply:
@skochay330 Thank you for the review I'll have a look at `EmptyList` :) But for some reason my latest changes aren't showing at all. This code is obsolete :(